scripts/check

```diff
@@ -18,12 +17,19 @@ configure_ssl_verification "${payload}"
 
 uri="$(jq -r '.source.uri // ""' < "${payload}")"
 private_token="$(jq -r '.source.private_token // ""' < "${payload}")"
+private_key="$(jq -r '.source.private_key // ""' < "${payload}")"
```

Comment:
Instead of checking for the `private_key`, I think it would logically be more correct to check the actual structure of the `uri`. I would rather do something like

```diff
+if [[ "${uri}" == "git@"* ]]; then
-if [[ ! -z "${private_key}" ]]; then
```

Reply:
Done